src/elements/content-preview/ContentPreview.scss

```diff
@@ -41,7 +41,7 @@
         position: absolute;
         top: 50%;
         transform: translateY(-50%);
-        transition-property: opacity;
+        transition: opacity .5s, left .3s cubic-bezier(.4, 0, .2, 1);
```

Comment:
The `opacity` entry here needs a timing function. Also, do we need to transition `right`, as well? Should they be the same duration?

Reply:
right should be ok because there's nothing that moves it on that side?

Reply:
Since this is in a compound rule, it's applied to the right arrow as well. Seems like it should live in the left arrow only, then? The duration and timing functions could live in the compound rule, while the properties get split up?